services/github/github-commit-activity.tester.js

```diff
@@ -11,6 +12,11 @@ const isCommitActivity = Joi.alternatives().try(
   isZeroOverTimePeriod
 )
 
+t.create('commit acticity (total)').get('/t/badges/shields.json').expectBadge({
```

Comment:
Typo here, but not sure if it breaks functionality?

Reply:
My bad, what bugs me is that CI was not able to catch it, I will take a look at it later.

Reply:
Yes it is just a test name, and I already merged a fix in https://github.com/badges/shields/commit/64d3380e1f83a7c2c9d05a80577980c2bcab1f31 no more follow-up needed